client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -113,6 +124,24 @@ const EventForm: React.FC<EventFormProps> = (props) => {
   const watchSponsorsArray = watch('sponsors');
   const inviteOnly = watch('invite_only');
   const venueType = watch('venue_type');
+  const chapter = watch('chapter_id');
```

Comment:
```suggestion
  const chapterId = watch('chapter_id');
```
Nit

Reply:
There's `chapterId` prop passed to the form. How does it sound renaming it in destructuring to ie. `initialChapterId` and using `chapterId` as in the suggestion?

Reply:
Ah, good point. Going with `initialChapterId` sounds perfect.